### ISSUE TITLE
High Loss

### ISSUE THREAD

**Issue by callmeToy** [pythonlessons/YOLOv3-object-detection-tutorial#9]
I am following the tutorial, i was trained with 6 class  (total 230 image 1280*720), the first  batchsize=32, num epoch = 50 and the second= 8,num epoch = 100, my loss is around 26-27,It doesn't come down . and when in epoch 75, it stopping with show text "early_stopping"
![Capture](https://user-images.githubusercontent.com/56773353/70691226-23b24780-1ceb-11ea-9f4d-2546e1c7b8ef.PNG)

**Comment by pythonlessons** [comment]
Hi, yes I know that loss is quite high always, but this is my past project,
I don't have time right now to come back to it, even with loss of 20 it has
quite high accuracy, just test it. When training loss will never go lover
than 20

**Comment by callmeToy** [comment]
I tried a lot of times with different epochs. but it all doesn't work. 
I wish you had more time.
thanks you !

**Comment by pythonlessons** [comment]
Try lower learning rate
